godtools/ViewControllers/Tract/TractViewControllerActions.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  TractViewControllerActions.swift
+//  godtools
+//
+//  Created by Pablo Marti on 6/6/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import MessageUI
+
+extension TractViewController: MFMailComposeViewControllerDelegate {
+    
+    func sendEmail(notification: Notification) {
+        if MFMailComposeViewController.canSendMail() {
+            guard let dictionary = notification.userInfo as? [String: Any] else {
+                return
+            }
+            
+            let composeVC = MFMailComposeViewController()
```

Comment:
This section and the else block below could be extracted into methods for good readability.